hooks/pre-commit/03-run-flake8

```diff
@@ -0,0 +1 @@
+/Users/yun.kim/dd/dd-trace-py/hooks/scripts/run-flake8.sh
```

Comment:
don't think will work super well on everyone's machines :)

Reply:
Crap lol I'm not sure how we can use symlinks here but it seems like it's required by autohook 😠